templates/zerver/footer.html

```diff
@@ -1,186 +1,108 @@
                 </a>
-            </li>
-            <li><a href="/api/">REST API</a></li>
-        </ul>
+                <a href="/api/">API</a>
+                <a href="https://github.com/zulip/zulip/">{{ _("GitHub") }}</a>
+            </p>
+        </div>
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("ABOUT US") }}
+            </span>
+            <p>
+                <span>
+                    <a href="/team/">{{ _("Team") }}</a>
+                    &
+                    <a href="/history/">{{ _("History") }}</a>
+                </span>
+                <a href="/values/">{{ _("Values") }}</a>
+                <a href="/jobs/">{{ _("Jobs") }}</a>
+                <a href="https://blog.zulip.com/"  target="_blank">{{ _("Blog") }}</a>
+                <a href="https://twitter.com/zulip/">Twitter</a>
+                <a href="https://zulip.com/help/support-zulip-project">{{ _("Support Zulip") }}</a>
+            </p>
+        </div>
     </div>
-    <div class="footer-section">
-        <h3>{{ _("About us") }}</h3>
-        <ul>
-            <li><a href="/team/">{{ _("Team") }}</a> &amp; <a href="/history/">{{ _("History") }}</a></li>
-            <li><a href="/values/">{{ _("Values") }}</a></li>
-            <li><a href="https://twitter.com/zulip/">Twitter</a></li>
-            <li><a href="/jobs/">{{ _("Jobs") }}</a></li>
-            <li><a href="/attribution">{{ _("Website attributions") }}</a></li>
-            <li><a href="https://github.com/sponsors/zulip">{{ _("Sponsor Zulip") }}</a></li>
-        </ul>
-    </div>
-</div>
-{% else %}
-<div class="footer self-hosted-footer">
-    <div class="footer-section">
-        <ul>
-            <li>
-                <a href="/help">{{ _("Help center") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <ul>
-            <li>
-                <a href="/policies">{{ _("Policies") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="break"></div>
-    <div class="footer-section powered-by-zulip">
-        <ul>
-            <li>
-                {% trans %}
-                Powered by <a href="https://zulip.com" class="coloured-text">Zulip</a>
-                {% endtrans %}
-            </li>
-        </ul>
+    {% endif %}
+    <div class="footer__legal {% if not corporate_enabled %}footer__legal_not_corporate{% endif %}">
+        <div class="footer__legal-container">
+            <div class="copyright">© Kandra Labs, Inc. (“Zulip”)</div>
+            <div class="footer__legal-spacer"></div>
+            {% if not corporate_enabled %}
+            <a href="/help">{{ _("Help center") }}</a>
+            {% endif %}
+            <a href="{{ root_domain_uri }}/terms/">{{ _("Terms of Service") }}</a>
+            <a href="{{ root_domain_uri }}/privacy/">{{ _("Privacy policy") }}</a>
+            <a href="https://zulip.com/attribution/">{{ _("Attributions") }}</a>
```

Comment:
We should rename this back to "Website attributions"; they're not for the product, just the website.

Reply:
Okay.